Examples/HelloWorld/CMakeLists.txt

```diff
@@ -6,3 +6,45 @@ target_link_libraries ( HelloWorld ${SimpleITK_LIBRARIES} )
 if ( WRAP_CSHARP )
   csharp_add_executable( HelloWorldCS SimpleITKCSharpManaged.dll ${CMAKE_CURRENT_SOURCE_DIR}/HelloWorld.cs )
```

Comment:
Using the `sitk_add_csharp_test macro below should take care of building the csharp code.